### DISCUSSION TITLE
Action running as queued and can't cancel it

### DISCUSSION THREAD

**Discussion by 26pawarp** [community/community#128032] [Actions]
### Select Topic Area

Bug

### Body

I have tried to cancel the action on my repo but it won't let me. Now it has stopped me from deploying my most recent changes. I don't know how to stop it so I am looking for any help.

**Comment by HimangshuRanjan** [comment]
I think u are dealing with git issue.
Use git status to see the current status of your repository. U can try the following steps-
1. If there are uncommitted changes, commit them using 'git commit -m "Your message"'.
2. If u want to cancel the last action, u can use 'git reset --hard HEAD^'.
3. If it's a specific branch you're working on, ensure you're on that branch using 'git checkout your_branch_name'.
4. Once everything looks good, try deploying your changes again

**Comment by gcruzdel** [comment]
We had this issue a couple of days ago, creating another commit triggered the workflow and the action worked again. 

But there seems to be a bug in Github that causes this scenario.

**Comment by wissam-natex** [comment]
Same. 
Githubstatus.com says they're currently facing an issue with actions.

For me, pushing a subsequent dummy commit did not trigger the action, so I'm just waiting for them to fix the bug

**Reply by grandmas-favorite** [reply]
Hi @wissam-natex!

Is this issue still occurring for you? There was an [incident with Actions](https://github.com/orgs/community/discussions/128173) recently that may have affected this functionality.

**Reply by wissam-natex** [reply]
Hello @grandmas-favorite,

It fixed itself shortly after I'd posted that comment yesterday.
The queued actions failed, but then all subsequent ones succeeded.

Thank you!

**Comment by joshua-temple** [comment]
Just so I don't miss all the fun, tagging my comment from the previously closed but related [topic](https://github.com/orgs/community/discussions/25570#discussioncomment-9930756).

I should note, that the force cancel doesn't work for me (500), and even though this particular GHA is a part of a concurrency-group (`cancel-in-progress:true`), the queued workflow does not cancel, and the new one continues on.

So good news.. we aren't blocked.  I just hope `Billable Time` doesn't account for the three weeks of being queued ;)
